sig-docs/localization-subproject/README.md

```diff
@@ -0,0 +1,104 @@
+# Localization Subproject
+
+The Kubernetes Localization Subproject is owned by [SIG-Docs](https://github.com/kubernetes/community/tree/master/sig-docs) and run by the [Subproject leads](#leadership).
+
+The Kubernetes documentation has been localized into serveral different languages. Each localization is owned by a team of individulas that speaks the language.  A localization team is a selfsufficent community responsible for prioritizing work, translating content, and reviewing and merging PRs into the kubernetes/website repo for their localization. 
+
+The Localization Subproject's aim is to provide a place for localization teams to share ideas and standardize processes across all localizations. This page covers documentation, processes, and roles for the Kubernetes Localization teams and the Localization Subproject.
+
+## Meetings
+
+Regular Localization Meeting: The first Monday of the month at 15:00 UTC (biweekly). [Convert Your Timezone](http://www.thetimezoneconverter.com/?t=15:00&tz=UTC)
+
+- [Meeting notes and Agenda](https://docs.google.com/document/d/1NwO1AN8Ea2zlK8uAdaDAKf1-LZDAFvSewIfrKqfl5No/edit#)
+- [Meeting recordings](https://www.youtube.com/playlist?list=PL69nYSiGNLP3b5hlx0YV7Lo7DtckM84y8)
+
+A localization team may also hold language specific meetings. If you are interested in attending a language specific meeting see the [Localization Teams](#localization-teams) section for details on how to contact each localization team.
+
+## Subproject Leadership
+
+- **Subproject lead:** [Abigail McCarthy](https://github.com/a-mccarthy)
+
+- **Emeritus leads:** [Brad Topol](https://github.com/bradtopol)
+
+## Contact
+
+- Slack: [#sig-docs-localizations](https://kubernetes.slack.com/archives/C0191RDKHU1) Each localiztion also has a language specific channel, noted [below](#localization-teams). 
+- Mailing list: 
+
+## Responsibilities
+
+The Localization Subproject lead is **responsible** for
+
+- Leading subproject meetings and taking notes
+- Driving collabrative discussions and decisions around localization processes and standards
+- Ensuring the communication of localization and SIG Docs policy changes to localization teams
+- Advocating for localization friendly practices within SIG Docs and broader Kubernetes community
+- Helping Localization teams understand SIG Docs processes, espcially when languages are starting out
```

Comment:
Side Note: This is something that we should actively advocate for, as well. Despite documentation being present, there's a lot of confusion around how to start & I think we'd benefit from a maintainer track session at KubeCon EU/NA reg. this by one of the localization team maintainers.

Reply:
huge +1 on this. Doing a maintainer track is a good idea. Is this something we can still submit for Kubecon EU in April?

Reply:
Yep, 27th January is our deadline, and I'd be happy to submit a nom for folks who are interested in presenting and have the bandwidth to do so during KubeCon

Reply:
@a-mccarthy I just checked with @natalisucks and @reylejano, and they're also on board with this suggestion. However, this struck me a little late. Otherwise, we could have floated this during yesterday's meeting. WDYT about floating this news in the #sig-docs-localization channel and letting one of us know reg. the nomination?

Reply:
@divya-mohan0209 yes, i will ask if there is anyone interested in sharing their experiences!